src/contractTest/java/uk/gov/hmcts/reform/wataskmanagementapi/consumer/wa/TaskManagerGetTaskConsumerTest.java

```diff
@@ -245,6 +246,17 @@ private DslPart createResponseForGetTaskForWa() {
                         )
                         .stringType("next_hearing_id", "nextHearingId")
                         .datetime("next_hearing_date", "yyyy-MM-dd'T'HH:mm:ssZ")
+                        .object("permissions", (value) -> {
```

Comment:
Depending on the role, we get back certain permissions. With contract test it will fail if the consumer has a permission not available on the provider?

Since this is the permission, should this reflect all the permissions for this particular task?